src/light.ts

```diff
@@ -0,0 +1,186 @@
+          entity.id,
+          new Map([
+            [uc.entities.Light.ATTRIBUTES.STATE, uc.entities.Light.STATES.OFF],
+            [uc.entities.Light.ATTRIBUTES.BRIGHTNESS, params && params.brightness ? params.brightness : 0]
+          ])
+        );
+      }
+      break;
+    case uc.entities.Light.COMMANDS.ON:
+      // params is optional! Use a default if not provided.
+      // A real lamp might store the last brightness value, otherwise the integration could also keep track of the last value.
+      uc.getConfiguredEntities().updateEntityAttributes(
+        entity.id,
+        new Map([
+          [uc.entities.Light.ATTRIBUTES.STATE, uc.entities.Light.STATES.ON],
+          [uc.entities.Light.ATTRIBUTES.BRIGHTNESS, params && params.brightness ? params.brightness : 127]
+        ])
+      );
+      uc.getConfiguredEntities().updateEntityAttributes(
+        "test_mediaplayer",
+        new Map([[uc.entities.MediaPlayer.ATTRIBUTES.VOLUME, 24]])
+      );
+      break;
+    case uc.entities.Light.COMMANDS.OFF:
+      uc.getConfiguredEntities().updateEntityAttributes(
+        entity.id,
+        new Map([
+          [uc.entities.Light.ATTRIBUTES.STATE, uc.entities.Light.STATES.OFF],
+          [uc.entities.Light.ATTRIBUTES.BRIGHTNESS, params && params.brightness ? params.brightness : 0]
+        ])
+      );
+      break;
+    default:
+      return uc.api_definitions.STATUS_CODES.NOT_IMPLEMENTED.toString();
+  }
+
+  return uc.api_definitions.STATUS_CODES.OK.toString();
+};
+
+// create a light entity
+// normally you'd create this where your driver exposed the available entities
+// The entity name can either be string (which will be mapped to english), or a Map with multiple language entries.
+const name = new Map([
+  ["de", "Mein Lieblingslicht"],
+  ["en", "My favorite light"]
+]);
+
+const attributes: Partial<Record<string, string>> = {
+  [uc.entities.Light.ATTRIBUTES.STATE]: uc.entities.Light.STATES.OFF,
+  [uc.entities.Light.ATTRIBUTES.BRIGHTNESS]: "0"
+};
+
+const lightEntity = new uc.entities.Light.Light("my_unique_light_id", name, {
+  features: [uc.entities.Light.FEATURES.ON_OFF, uc.entities.Light.FEATURES.DIM],
+  attributes
+});
+lightEntity.setCmdHandler(lightCmdHandler ?? null);
+
+// add entity as available
+// this is important, so the core knows what entities are available
+uc.getAvailableEntities().addEntity(lightEntity);
+
+const buttonEntity = new uc.entities.Button.Button("my_button", "Push the button!", {
+  area: "test lab",
+  cmdHandler: sharedCmdHandler
+});
+uc.getAvailableEntities().addEntity(buttonEntity);
+
+const defaultAttributes: Partial<Record<string, string | string[]>> = {
+  [uc.entities.MediaPlayer.ATTRIBUTES.STATE]: uc.entities.MediaPlayer.STATES.ON,
+  [uc.entities.MediaPlayer.ATTRIBUTES.SOURCE_LIST]: ["Radio", "Streaming", "Favorite 1", "Favorite 2", "Favorite 3"]
+};
+
+// add a media-player entity
+const mediaPlayerEntity = new uc.entities.MediaPlayer.MediaPlayer(
```

Comment:
Is it possible to use the same default exports in TS as previously in JS or something similar to avoid the double entity names?
- new uc.entities.Light.Light
- new uc.entities.Button.Button
- new uc.entities.MediaPlayer.MediaPlayer

Reply:
It is fixed now, no more double entity names.